README.md

```diff
@@ -76,10 +83,8 @@ this means:
 
 # Getting Started
 
-* [Quickstart](Documentation/quickstart.md)
-* [Installation](Documentation/installation.md)
-* [Documentation](Documentation/README.md)
-* [Tutorials](tutorials/README.md)
+* [Networking and Security Observability with Hubble](https://docs.cilium.io/en/latest/gettingstarted/hubble/)
+* [Introduction to Cilium & Hubble](https://docs.cilium.io/en/latest/intro/)
```

Comment:
Maybe swap these two lines? It makes more sense to have the "Introduction to Cilium & Hubble" link first.